testing/streamlink/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Robert Sacks <robert@sacks.email>
+# Maintainer: Robert Sacks <robert@sacks.email>
+pkgname="streamlink"
+pkgver="1.1.1"
+pkgrel=0
+pkgdesc="CLI for extracting streams from various websites to a video player of
+	your choosing"
+url="https://streamlink.github.io/"
+arch="all"
+license="BSD-2-Clause"
+depends="python3 py3-pycryptodome py3-pycountry py3-requests py3-pysocks
+	py3-websocket-client py3-isodate rtmpdump"
+makedepends="py3-setuptools py3-sphinx py3-recommonmark py3-pytest py3-mock
+	py3-freezegun py3-requests-mock"
```

Comment:
I think some of these belong into checkdepends :)

Reply:
Fixed